src/Area.ts

```diff
@@ -52,9 +61,9 @@ export class Area extends GameEntity {
 	/** Active NPCs that originate from this area. Note: this is NPCs that */
 	npcs: Set<Npc>;
 	metadata: Metadata;
-  behaviors: Map<string, any>;
+	behaviors: Map<string, any>;
 
-	constructor(bundle: string | null, name: string, manifest: IAreaDef) {
+	constructor(bundle: string | null, name: string, manifest: IAreaManifest) {
```

Comment:
There is a distinction between the manifest and the definition. The definition holds the manifest and has the bundle property on it. The definition is also the type that is stored in the AreaFactory.

Reply:
In my own repo, I did flatten Areadef and AreaManifest to be the same thing, was less confusing that way though.